criu/config.c

```diff
@@ -765,13 +777,17 @@ int parse_options(int argc, char **argv, bool *usage_error,
 					goto bad_arg;
 
 				*aux = '\0';
-				if (ext_mount_add(optarg, aux + 1))
+				if (ext_mount_add(optarg, aux + 1)) {
+					pr_err("Could not xmalloc when initializing config.\n");
```

Comment:
Please don't include information about implementation details outside of the function. Although it might be true now, that could change in the future and the person editing the body of that function won't know about that message here. (I'm talking about the `xmalloc` part)